.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Lint and Test
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+      - master
+      - ci-workflow
+
+env:
+  GO_VERSION: stable
+  GOLANGCI_LINT_VERSION: v1.60
+
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: ${{ env.GOLANGCI_LINT_VERSION }}
+          only-new-issues: true
```

Comment:
did this config change at some point? I just see `new` in these docs - https://golangci-lint.run/usage/configuration/

Reply:
Honestly I couldn't figure this one out - `only-new-issues` seems to work, but it's a little hard to test

Reply:
Haha well I guess it works... I knew this would be an issue, do you want me to fix in this PR or start a new one to deal with the issues?